### PR TITLE
feat: Personal Space , Sessions Dashboard , Log out

### PR DESCRIPTION
**Pull request by avevotsira**

### CODE REVIEW

**Comment by avevotsira** [comment]
image
image

File: src/app/(session)/sessions/page.tsx [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+interface SessionsResponse {
+  data: Session[];
+}
+
+export default function SessionsPage() {
+  const [sessionToRevoke, setSessionToRevoke] = useState<string | null>(null);
+  const { data: sessionsResponse, error } = useSWR<SessionsResponse>(
+    "/v1/auth/sessions",
+    fetcher
+  );
```
Comment (invisal) [inline src/app/(session)/sessions/page.tsx:36]: It is better to follow our current pattern of using hook. Like `useSessionList()`

File: src/components/containers/personal-container.tsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+          SessionId: token,
+        },
+      });
+
+      localStorage.removeItem("token");
+      router.push("/login");
+    } catch (error) {
+      console.error(error);
+    }
+  };
```
Comment (invisal) [inline src/components/containers/personal-container.tsx:43]: Can we turn it into a hook like ```useRevokeToken```

**Review by invisal — CHANGES_REQUESTED** [review]

**Review by invisal — APPROVED** [review]
LGTM

**Pull request merged**